Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -105,6 +105,13 @@ configuration:
   name: use_requests_handler
   required: false
   type: 8
+- defaultvalue: '1'
+  display: The interval in minutes to fetch incidents. (e.g. 5 would fetch every five
```

Comment:
we will need better explanation to this interval as it go backwards in 5 minutes.